stats/index.js

```diff
@@ -14,7 +14,7 @@ for (var j = 0; j < 100; ++j) {
   for (var i = 1; i < 4; ++i) {
     let utxos = Simulation.generateTxos(20 / i, min, max)
     let txos = Simulation.generateTxos(80 / i, min, max / 3)
-
+    txos.forEach(txo => txo.script.length = 34)
```

Comment:
It's the same issue that I solved in the other simulation here
https://github.com/bitcoinjs/coinselect/pull/15/commits/96948ce6a6ae89f54850e78ff3b252076fbac6a2

(script.length cannot be empty, otherwise simulation thinks it's change and adds it back to utxos)